test/d_do_test.d

```diff
@@ -565,6 +569,10 @@ int tryMain(string[] args)
     if (!gatherTestParameters(testArgs, input_dir, input_file, envData))
         return 0;
 
+    // Clear the DFLAGS environment variable if it was specified in the test file
+    if (testArgs.noDflags)
+        environment.remove("DFLAGS");
```

Comment:
Shouldn't this set `-conf=` too? Otherwise the DFLAGS from the dmd.conf would be used.